### PR TITLE
Docs: Add "Recommended Reading" to the docs homepage

### LINKED ISSUE #729: Docs: Update docs homepage with improved links and CTAs
### Description

Docs home page was built as a very quick first iteration, and with current traffic, feel like we could provide better information and more useful resources to the users visiting the site. This is our PH heatmap for the past 7 days:

Screenshot 2024-04-02 at 17 21 32

**Proposal:**

- Remove the three tiles and replace them with "Recommended Reading" & move "Diverse Widget Collection" to be a third pill CTA
- Add CTAs for FlowFuse below this to help people get started with a NR Instance quickly.

### Epic/Story

_No response_

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- Removes the default vitepress "Feature" tiles
- Adds new "Recommended Reading" section
- Adds "Available in Node-REDs Palette Manager" option, with quick copy button

## Related Issue(s)

Closes #729

### CODE REVIEW

**Comment by joepavitt** [comment]
### Before

Screenshot 2024-04-02 at 17 31 29

### After

Screenshot 2024-04-02 at 17 31 09

**Comment by ZJvandeWeg** [comment]
Quite clean, will check it out locally too.

File: docs/index.md [hunk 1]
```diff
@@ -12,27 +12,68 @@ hero:
...
+---
+
+<script setup>
+    import { ref } from 'vue';
+
+    import HomeExtension from './components/HomeExtension.vue';
+    import RecommendedReading from './components/RecommendedReading.vue';
+    import CopyIcon from './components/icons/CopyIcon.vue';
+
+    const copied = ref(false);
```
Comment (ZJvandeWeg) [inline docs/index.md:30]: Just to validate; `copied` can be `const` as it's the reference to false. Than the value is later updated to `true`. 

Why not just `let copied = false` here?

**Comment by ZJvandeWeg** [comment]
Surprisingly small diff, nice work @joepavitt

**Review by ZJvandeWeg — APPROVED** [review]

File: docs/index.md (hunk 1 above)
Comment (joepavitt) [reply docs/index.md:30]: using the `ref` is a way of making it accessible to the HTML/template, eqv of `data () { ... }` in a full vue component definition

**Pull request merged**